common/constants/shared.ts

```diff
@@ -35,6 +35,7 @@ export const JOBS_ENDPOINT_BASE = '/_plugins/_async_query';
 export const JOB_RESULT_ENDPOINT = '/result';
 
 export const tutorialSampleDataPluginId = 'import_sample_data';
+export const dataSourcesId = 'dataSources';
```

Comment:
Can we please name this as `dataSourceManagementPluginId`?

Reply:
Changed.